packages/ui/src/theme/tokens/components/select.ts

```diff
@@ -24,20 +28,27 @@ export type SelectTokens<Output extends OutputVariantKey> =
     >;
     iconWrapper?: SelectIconWrapperTokens<Output>;
     option?: DesignTokenProperties<'backgroundColor' | 'color', Output> & {
-      _disabled?: DesignTokenProperties<'color', Output>;
+      _disabled?: DesignTokenProperties<'color' | 'backgroundColor', Output>;
     };
     small?: SelectSizeTokens<Output>;
     large?: SelectSizeTokens<Output>;
-    multiple?: DesignTokenProperties<
+    expanded?: DesignTokenProperties<
       'paddingBlock' | 'paddingInline',
       Output
     > & {
       option?: DesignTokenProperties<'paddingBlock' | 'paddingInline', Output>;
     };
+    _disabled?: DesignTokenProperties<'backgroundColor' | 'color', Output>;
   };
 
 export const select: Required<SelectTokens<'default'>> = {
+  color: { value: '{components.fieldcontrol.color}' },
+  backgroundColor: { value: '{colors.background.primary.value}' },
```

Comment:
Firefox on Windows plays nicer with option colors/background when a background is set on the select itself. Otherwise, you get these top and bottom "borders" (screenshot below). So I added new backgroundColor/color tokens for select. I've set the existing backgroundColor tokens for option to be transparent. I added a color token to pair it with to make sure we/developers can always provide a sufficient contrast.

![Screenshot (813)](https://github.com/aws-amplify/amplify-ui/assets/376920/0be27666-5512-4e8a-b003-7eaf5fb12881)